test/test_helper.rb

```diff
@@ -1,6 +1,7 @@
 ENV["RAILS_ENV"] = "test"
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
+require 'capybara/rails'
```

Comment:
We are not using `test/test_helper.rb`, this is default Rails stack. In this course, we are using RSpec and `spec_helper.rb`. I would recommend you to move the setup for specs there.